gst/nnstreamer/nnstreamer_plugin_api_util_impl.c

```diff
@@ -309,9 +309,13 @@ gst_tensor_info_get_rank (const GstTensorInfo * info)
 GstTensorInfo *
 gst_tensors_info_get_nth_info (GstTensorsInfo * info, guint nth)
 {
+  g_return_val_if_fail (info != NULL, NULL);
+
   if (nth < NNS_TENSOR_SIZE_LIMIT)
     return &info->info[nth];
 
+  gst_tensors_info_extra_create (info);
```

Comment:
How about using return value of `gst_tensors_info_extra_create`?
```suggestion
  if (!gst_tensors_info_extra_create (info)) {
     return NULL;
 }
```

Reply:
Agreed, I will update this after merging previous PR.

Reply:
updated. plz review the changes again.